Source/Project64/main.cpp

```diff
@@ -100,4 +101,4 @@ int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, LPSTR /
     AppCleanup();
     CoUninitialize();
     return true;
-}
\ No newline at end of file
+}
```

Comment:
While we are here, can we make sure the files have an empty new line at the end so every change to the file doesn't have this "change"?